### ISSUE TITLE
TextEditor.setScrollTop is deprecated.

### ISSUE THREAD

**Issue by floriandierickx** [dundalek/atom-markdown-mindmap#17]
This is now a view method. Call TextEditorElement::setScrollTop instead.
```
TextEditor.setScrollTop (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:3494:12)
<unknown> (.../.atom/packages/markdown-mindmap/lib/markdown-mindmap-view.coffee:312:23)
```

**Comment by floriandierickx** [comment]
Everything is working perfectly fine (thanks a lot for that!), but I got this error message popping up recently.

**Comment by dundalek** [comment]
Thanks for reporting, I will look into it.

**Comment by dundalek** [comment]
This is now fixed in the `v0.3.3` update.
